src/cards/attack/attack_card_data.gd

```diff
@@ -10,4 +10,4 @@ extends Resource
 @export var attack_power: int ## The amount of damage this card does each turn.
 @export var attack_recoil: int ## The amount of damage this card takes after attacking.
 @export var health: int ## The card's max health.
-@export_file() var unit ## A path the the unit this card will spawn.
+@export_file() var unit ## A path to the unit this card will spawn.
```

Comment:
We might be able to specify:

```gdscript
@export_file("res://src/units/*.tscn") var unit
```

Or something along those lines to specify spawn units. Not needed now but would be cool